modules/Cockpit/assets/components/field-select.tag

```diff
@@ -2,7 +2,7 @@
 
     <select ref="input" class="uk-width-1-1 {opts.cls}" bind="{ opts.bind }">
         <option value=""></option>
-        <option each="{ option,idx in options }" value="{ option.value }" selected="{ parent.root.$value === option.value }">{ option.label }</option>
+        <option each="{ option,idx in options }" value="{ option.value }" selected="{ parent.root.$value == option.value }">{ option.label }</option>
```

Comment:
what is the reason for this change?

Reply:
This is required as field-select values are casted to strings , while sort direction is a number.
With strict comparison sort direction would have to be string

Reply:
Here I'm converting values from `field-select` back to integer: https://github.com/agentejo/cockpit/pull/1246/files#diff-9f7295b64abd3cc5e2d21b9b01b75af2R302-R305

It doesn't feel completely correct, I admit

Reply:
If you'd like to keep strict value comparison in field-select, another option is to convert direction to string specially for the field-select and than cast back, as here: https://github.com/piotr-cz/cockpit/commit/b5a481003619d8653ce84c43313dea04792c6e49

Maybe there is better way but I'm not familiar with Riot.js